recipes/stoken/yum_requirements.txt

```diff
@@ -0,0 +1,9 @@
+libX11
+libXcomposite
+libXcursor
+libXdamage
+libXext
+libXfixes
+libXi
+libXinerama
+libXrandr
```

Comment:
These should have CDTs

Reply:
I don't think it's necessary to add these as CDTs, because they aren't necessary for building.  They are run-time dependencies of `gtk3`, though.  There is an [ongoing discussion regarding how gtk3 dependencies][1] should handle these, but for now we are simply listing them in `yum_requirements.txt`.

[1]: https://github.com/conda-forge/gtk3-feedstock/issues/7

Reply:
If they are only run time deps, do you need them at all?

Reply:
I'm trying to remember the error... I think the DSO checks failed somehow.  Let me temporarily remove `yum_requirements.txt` to reproduce the error.

Reply:
Oh wait, it's mentioned in that linked discussion.  It turns out that `gtk3` has a `post-link.sh` script, and that script fails if some of the dependencies are missing.

So the failure occurs in this feedstock during the test phase.

Reply:
Yep, the failure is in the `gtk3` post-link script:

<details>

<summary>post-link error</summary>

```
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
ERROR conda.core.link:_execute(700): An error occurred while installing package 'conda-forge::gtk3-3.24.18-h45fd312_1'.
Rolling back transaction: ...working... done
Traceback (most recent call last):
  File "/home/conda/.ci_support/build_all.py", line 133, in <module>
    build_all(args.recipes_dir, args.arch)
  File "/home/conda/.ci_support/build_all.py", line 66, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File "/home/conda/.ci_support/build_all.py", line 123, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2864, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1911, in build
    create_build_envs(top_level_pkg, notest)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1751, in create_build_envs
    is_conda=m.name() == 'conda')
  File "/opt/conda/lib/python3.7/site-packages/conda_build/environ.py", line 875, in create_env
    execute_actions(actions, index)
  File "/opt/conda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/opt/conda/lib/python3.7/site-packages/conda/plan.py", line 321, in execute_actions
    execute_instructions(plan, index, verbose)
  File "/opt/conda/lib/python3.7/site-packages/conda/plan.py", line 533, in execute_instructions
    cmd(state, arg)
  File "/opt/conda/lib/python3.7/site-packages/conda/instructions.py", line 73, in UNLINKLINKTRANSACTION_CMD
    unlink_link_transaction.execute()
  File "/opt/conda/lib/python3.7/site-packages/conda/core/link.py", line 249, in execute
    self._execute(tuple(concat(interleave(itervalues(self.prefix_action_groups)))))
  File "/opt/conda/lib/python3.7/site-packages/conda/core/link.py", line 716, in _execute
    rollback_excs,
conda.CondaMultiError: post-link script failed for package conda-forge::gtk3-3.24.18-h45fd312_1
location of failed script: /home/conda/staged-recipes/build_artifacts/stoken_1591127839206/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/bin/.gtk3-post-link.sh
==> script messages <==
<None>
==> script output <==
stdout: 
stderr: /home/conda/staged-recipes/build_artifacts/stoken_1591127839206/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/bin/gtk-query-immodules-3.0: error while loading shared libraries: libXi.so.6: cannot open shared object file: No such file or directory

return code: 127

```

</details>